src/components/start-process/start-process.tsx

```diff
@@ -0,0 +1,81 @@
+import { Link as RouterLink, useHistory } from "react-router-dom";
+
+import { Form, Formik } from "formik";
+import * as Yup from "yup";
+
+import { locale } from "../../services";
+
+import { Button, Checkbox, Link } from "@mtfh/common/lib/components";
+
+import "./styles.scss";
+
+const { components } = locale;
+
+interface StartProcessProps {
+  process: {
+    thirdPartyCondition: string;
+    ThirdPartyContent: any;
```

Comment:
Is this a React component? Can we have stronger typing here?

Reply:
Ah yes, this was meant to be temporary.